example-1/styles.css

```diff
@@ -0,0 +1,100 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
+    Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  font-size: 16px;
+  background: #e6f1ff;
+}
+
+.wrapper {
+  display: flex;
+  margin: 40px auto 0;
+  width: 80%;
+  max-width: 718px;
+}
+
+.wrapper form {
+  width: 100%;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+}
+
+label {
+  opacity: 0.6;
+  font-size: 18px;
+  padding-bottom: 7px;
+  padding-top: 13px;
+  font-weight: 500;
+  display: block;
+}
+
+.mollie-component {
+  background: #ffffff;
+  box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.1), 0px 2px 4px rgba(0, 0, 0, 0.1),
+    0px 4px 8px rgba(0, 0, 0, 0.05);
+  border-radius: 4px;
+  padding: 13px;
+  border: 1px solid transparent;
+  transition: 0.15s border-color cubic-bezier(0.4, 0, 0.2, 1);
+  color: rgba(0, 0, 0, 0.8);
```

Comment:
dont think this does something if you want to change the color of the input fieldsyou should give that as an option to the `Mollie.createComponent([name], {style:{color: rgba(0,0,0,0.8) }} )`